lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -17,12 +17,29 @@ module FlashHelper
     }
     def display_flash_messages(key_matching = {})
       key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
+      key_matching.default = :standard
 
-      flash.inject "" do |message, (key, value)|
-        message += content_tag :div, :data => { :alert => "" }, :class => "alert-box #{key_matching[key.to_sym] || :standard}" do
-          (value + link_to("&times;".html_safe, "#", :class => :close)).html_safe
+      capture do
+        flash.each do |key, value|
+          next if FoundationRailsHelper.configuration.ignored_flash_keys.include? key.to_sym
```

Comment:
Sorry I wasn't clear. Your approach is the way to go.  And I agree that changing the parameter to a config should be a separate PR.  Maybe instead of a breaking change, we allow both ways for one release and deprecate the paramater.  Then remove the parameter on the second release.